gateway/api/runbooks/rules.go

```diff
@@ -0,0 +1,209 @@
+package apirunbooks
+
+import (
+	"database/sql"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"github.com/hoophq/hoop/common/log"
+	"github.com/hoophq/hoop/gateway/api/openapi"
+	"github.com/hoophq/hoop/gateway/models"
+	"github.com/hoophq/hoop/gateway/storagev2"
+)
+
+// ListRunbookRules
+//
+// @Summary      List Runbook Rules
+// @Description  List all Runbook Rules
+// @Tags         Runbooks
+// @Produce      json
+// @Success      200  {object}  []openapi.RunbookRule
+// @Failure      404,422,500  {object} openapi.HTTPError
+// @Router       /runbooks/rules [get]
+func ListRunbookRules(c *gin.Context) {
+	ctx := storagev2.ParseContext(c)
+
+	rules, err := models.GetRunbookRules(models.DB, ctx.GetOrgID(), 0, 0)
+	if err != nil {
+		log.Infof("failed fetching runbook rules, err=%v", err)
+		c.JSON(http.StatusUnprocessableEntity, gin.H{"message": "failed fetching runbook rules"})
+		return
+	}
+
+	responseRules := make([]openapi.RunbookRule, 0, len(rules))
+	for _, rule := range rules {
+		responseRules = append(responseRules, buildRunbookResponseFromModel(&rule))
+	}
+
+	c.JSON(http.StatusOK, responseRules)
+}
+
+// GetRunbookRule
+//
+// @Summary      Get Runbook Rule
+// @Description  Get a single Runbook Rule by ID
+// @Tags         Runbooks
+// @Produce      json
+// @Param        id  path      string  true  "Runbook Rule ID"
+// @Success      200  {object}  openapi.RunbookRule
+// @Failure      404,422,500  {object} openapi.HTTPError
+// @Router       /runbooks/rules/{id} [get]
+func GetRunbookRule(c *gin.Context) {
+	ctx := storagev2.ParseContext(c)
+
+	ruleID := c.Param("id")
+
+	rule, err := models.GetRunbookRuleByID(models.DB, ctx.GetOrgID(), ruleID)
+	if err != nil {
+		log.Infof("failed fetching runbook rule, err=%v", err)
```

Comment:
Do not print Info messages to not found errors, only internal errors or something that you really want to know that happened. Use debugf if you want to keep this line.